src/test/librbd/test_ImageWatcher.cc

```diff
@@ -214,6 +214,15 @@ class TestImageWatcher : public TestFixture {
       {
         RebuildObjectMapPayload payload;
         payload.decode(2, iter);
+        cout << payload.async_request_id;
```

Comment:
i'd avoid writing to `cout` in tests, unless something unexpected happens.

Reply:
Yes. Historically it is not always true for all our unit tests, but I would not add it in the new ones. You can get this info running the test with CEPH_ARGS=--debug_rbd=20 from the ceph client log.